Dockerfile

```diff
@@ -1,3 +1,55 @@
+FROM alpine:latest as BUILDER
+ARG NAME=consul-ecs
+ARG VERSION
+
+# This is the location of the releases.
+ENV HASHICORP_RELEASES=https://releases.hashicorp.com
+
+# Set up certificates, base tools, and software.
+RUN set -eux && \
+    apk add --no-cache ca-certificates curl gnupg libcap openssl su-exec iputils libc6-compat iptables && \
```

Comment:
do we need all of these in the builder image? Probably don't need su-exec, iputils, iptables

Reply:
oh yeah I just copied that over, let me minimize this...